echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/impl/MissedPipelineTriggerCompensationJob.groovy

```diff
@@ -199,14 +200,14 @@ class MissedPipelineTriggerCompensationJob implements ApplicationListener<Contex
       }
 
       try {
-        CronExpression expr = new CronExpression(trigger.cronExpression)
+        CronExpression expr = getCronExpression(trigger)
         expr.timeZone = dateContext.timeZone
 
         if (missedExecution(expr, lastExecution, dateContext.triggerWindowFloor, dateContext.now, pipeline)) {
           pipelineInitiator.call(pipeline)
         }
       } catch (ParseException e) {
-        log.error("Error parsing cron expression (${trigger.cronExpression}) for pipeline ${pipeline.id}")
+        log.error("Error parsing cron expression (${trigger.cronExpression}) for pipeline ${pipeline.id}", e)
```

Comment:
Don't think logging a stacktrace here is worth the bits.

Reply:
At least the error message then if not the full stack trace? I have a burning hatred for error logs that don't show the actual exception